core/jazz_metrics/index.js

```diff
@@ -24,22 +24,13 @@ const errorHandlerModule = require("./components/error-handler.js"); //Import th
 const responseObj = require("./components/response.js"); //Import the response module.
 const configObj = require("./components/config.js"); //Import the environment data.
 const logger = require("./components/logger.js")(); //Import the logging module.
-const aws = require("aws-sdk"); //Import the secret-handler module.
 const request = require('request');
 const utils = require("./components/utils.js"); //Import the utils module.
 const validateUtils = require("./components/validation.js");
-const global_config = require("./config/global-config.json");
-
-module.exports.handler = (event, context, cb) => {
 
+function handler (event, context, cb) {
   var errorHandler = errorHandlerModule();
   var config = configObj(event);
-  var cloudwatch = new aws.CloudWatch({
```

Comment:
Looks like metrics is using older config reader, can you update it?

Reply:
Done